### PR TITLE
completed html forms

### PR DESCRIPTION
**Pull request by salgarcia666**
Screenshot (319)
sal garcia: completed html forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<label for="js">JavaScript </label>
+<div>
+<div>Would you like to sign up for more information?</div>
+<input type="checkbox">
+<label for="checkbox">Sign me up!</label>
+</div>
+<div>
+  <label for="text">Comments</label>
+  <div>
+  <input type="text">
```
Comment (Zhiwutian) [inline html-forms/index.html:48]: this should be a textarea input

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update to a text area. And I would make sure all of the form elements are properly indented in the form.

File: html-forms/index.html (hunk 1 above)
Comment (salgarcia666) [reply html-forms/index.html:48]: Screenshot (319)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**